src/App.tsx

```diff
@@ -1,177 +1,15 @@
 import React from 'react';
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+
+import { MoviesList } from './components/MoviesList';
 
 /* eslint-disable max-len */
 export const App: React.FC = () => (
-  <div className="page">
+  <div className="page" data-cy="card">
```

Comment:
Are you sure that you need this data-cy here?

Reply:
I don't remember what it's for anymore🥲